api/rpc/perms/permissions.go

```diff
@@ -28,9 +32,16 @@ func (j *JWTPayload) MarshalBinary() (data []byte, err error) {
 
 // NewTokenWithPerms generates and signs a new JWT token with the given secret
 // and given permissions.
-func NewTokenWithPerms(signer jwt.Signer, perms []auth.Permission) ([]byte, error) {
+func NewTokenWithPerms(signer jwt.Signer, perms []auth.Permission, ttl time.Duration) ([]byte, error) {
+	var nonce [32]byte
```

Comment:
Is there particular reason to use fixed sized array here? if slice is used instead of array it will remove convertions by [:]

Reply:
ok, fixed.